assets/data_table/main.js

```diff
@@ -421,6 +426,16 @@ function App({ ctx, data }) {
     });
   }, []);
 
+  useEffect(() => {
```

Comment:
We can have all subscriptions in the same useEffect

```suggestion
    });

```